### ISSUE TITLE
Errors in prepareData

### ISSUE THREAD

**Issue by agilly** [hmgu-itg/variant_annotation#1]
When running `prepareData` with `-o` only, the following errors arise:

```bash
13/10/2020 05:51:52 Creating GTEx file
ERROR: no gene ID in row 437520
ERROR: no gene ID in row 437524
ERROR: no gene ID in row 437537
ERROR: no gene ID in row 437549
ERROR: no gene ID in row 437553
ERROR: no gene ID in row 437569
ERROR: no gene ID in row 437573
ERROR: no gene ID in row 437576
ERROR: no gene ID in row 437580
ERROR: no gene ID in row 437583
ERROR: no gene ID in row 437586
ERROR: no gene ID in row 437595
ERROR: no gene ID in row 437599
ERROR: no gene ID in row 437604
ERROR: no gene ID in row 437616
ERROR: no gene ID in row 437637
ERROR: no gene ID in row 437640
ERROR: no gene ID in row 437643
ERROR: no gene ID in row 437657
ERROR: no gene ID in row 437664
ERROR: no gene ID in row 437670
ERROR: no gene ID in row 437676
ERROR: no gene ID in row 437692
ERROR: no gene ID in row 437697
ERROR: no gene ID in row 437704
ERROR: no gene ID in row 437715
ERROR: no gene ID in row 437721
ERROR: no gene ID in row 437725
ERROR: no gene ID in row 437739
ERROR: no gene ID in row 437743
ERROR: no gene ID in row 437746
ERROR: no gene ID in row 437749
ERROR: no gene ID in row 437762
ERROR: no gene ID in row 437766
ERROR: no gene ID in row 440905
ERROR: no gene ID in row 440909
ERROR: no gene ID in row 440912
ERROR: no gene ID in row 440915
ERROR: no gene ID in row 440925
ERROR: no gene ID in row 440928
ERROR: no gene ID in row 440931
ERROR: no gene ID in row 440944
ERROR: no gene ID in row 440948
ERROR: no gene ID in row 440953
Adipose_Subcutaneous
GTEx_Analysis_v8_eQTL/Adipose_Subcutaneous.v8.signif_variant_gene_pairs.txt.gz
Adipose_Visceral_Omentum
GTEx_Analysis_v8_eQTL/Adipose_Visceral_Omentum.v8.signif_variant_gene_pairs.txt.gz
Adrenal_Gland
```` 

These errors should be corrected, or if benign, silenced.

**Comment by agilly** [comment]
Adding `print(row)` at l.78 in `prepareGTEx.py` shows that all these genes are of the form `ENSGXXXXX.XX_PAR_Y`. These are duplicate genes on the pseudoautosomal chromosome Y.

**Comment by agilly** [comment]
fixed by a851b112e406d3e4dd38cf3f47ac3cd559de1934
